src/scribe_data/resources/language_metadata.json

```diff
@@ -1,70 +1,182 @@
+  "latin": {
+    "iso": "la",
+    "qid": "Q397"
+  },
+  "malay": {
+    "iso": "ms",
+    "qid": "Q9237"
+  },
+  "malayalam": {
+    "iso": "ml",
+    "qid": "Q36236"
+  },
+  "chinese": {
+    "sub_languages": {
+      "mandarin": {
+        "iso": "zh",
+        "qid": "Q727694"
+      }
+    }
+  },
+  "norwegian": {
+    "sub_languages": {
+      "nynorsk": {
+        "iso": "nn",
+        "qid": "Q25164"
+      },
+      "bokmål": {
+        "iso": "nb",
+        "qid": "Q25167"
+      }
+    }
+  },
+  "pidgin": {
+    "sub_languages": {
+      "nigerian": {
+        "iso": "pi",
+        "qid": "Q33655"
+      }
     }
   },
-  "languages": [
-    {
-      "language": "english",
-      "iso": "en",
-      "qid": "Q1860",
-      "remove-words": ["of", "the", "The", "and"],
-      "ignore-words": []
-    },
-    {
-      "language": "french",
-      "iso": "fr",
-      "qid": "Q150",
-      "remove-words": ["of", "the", "The", "and"],
-      "ignore-words": ["XXe"]
-    },
-    {
-      "language": "german",
-      "iso": "de",
-      "qid": "Q188",
-      "remove-words": ["of", "the", "The", "and", "NeinJa", "et", "redirect"],
-      "ignore-words": ["Gemeinde", "Familienname"]
-    },
-    {
-      "language": "italian",
-      "iso": "it",
-      "qid": "Q652",
-      "remove-words": ["of", "the", "The", "and", "text", "from"],
-      "ignore-words": ["The", "ATP"]
-    },
-    {
-      "language": "portuguese",
-      "iso": "pt",
-      "qid": "Q5146",
-      "remove-words": ["of", "the", "The", "and", "jbutadptflora"],
-      "ignore-words": []
-    },
-    {
-      "language": "russian",
-      "iso": "ru",
-      "qid": "Q7737",
-      "remove-words": ["of", "the", "The", "and"],
-      "ignore-words": []
-    },
-    {
-      "language": "spanish",
-      "iso": "es",
-      "qid": "Q1321",
-      "remove-words": ["of", "the", "The", "and"],
-      "ignore-words": []
-    },
-    {
-      "language": "swedish",
-      "iso": "sv",
-      "qid": "Q9027",
-      "remove-words": ["of", "the", "The", "and", "Checklist", "Catalogue"],
-      "ignore-words": ["databasdump"]
+  "polish": {
+    "iso": "pl",
+    "qid": "Q809"
+  },
+  "portuguese": {
+    "iso": "pt",
+    "qid": "Q5146"
+  },
+  "punjabi": {
+    "sub_languages": {
+      "shahmukhi": {
+        "iso": "pnb",
+        "qid": "Q58635"
+      },
+      "gurmukhi": {
+        "iso": "pa",
+        "qid": "Q58635"
+      }
     }
-  ]
+  },
+  "russian": {
+    "iso": "ru",
+    "qid": "Q7737"
+  },
+  "slovak": {
+    "iso": "sk",
+    "qid": "Q9058"
+  },
+  "spanish": {
+    "iso": "es",
+    "qid": "Q1321"
+  },
+  "swahili": {
+    "iso": "sw",
+    "qid": "Q7838"
+  },
+  "swedish": {
+    "iso": "sv",
+    "qid": "Q9027"
+  },
+  "tajik": {
+    "iso": "tg",
+    "qid": "Q9260"
+  },
+  "tamil": {
+    "iso": "ta",
+    "qid": "Q5885"
+  },
+  "ukrainian": {
+    "iso": "ua",
+    "qid": "Q8798"
+  },
+  "yoruba": {
+    "iso": "yo",
+    "qid": "Q34311"
+  }
 }
```

Comment:
Thanks for cleaning the `language_metadata.py`. It is now much easier to look upto

Reply:
Thank you for your feedback! I'm glad to hear that the JSON file is easier to work with now. I also want to thank @andrewtavis for his guidance on this.